### PR TITLE
hdl-dump: init at 20202807

### PR DESCRIPTION
**Pull request by AndersonTorres**
###### Things done

- [ ] Tested using sandboxing ([nix.useSandbox](https://nixos.org/nixos/manual/options.html#opt-nix.useSandbox) on NixOS, or option `sandbox` in [`nix.conf`](https://nixos.org/nix/manual/#sec-conf-file) on non-NixOS linux)
- Built on platform(s)
   - [ ] NixOS
   - [ ] macOS
   - [ ] other Linux distributions
- [ ] Tested via one or more NixOS test(s) if existing and applicable for the change (look inside [nixos/tests](https://github.com/NixOS/nixpkgs/blob/master/nixos/tests))
- [ ] Tested compilation of all pkgs that depend on this change using `nix-shell -p nixpkgs-review --run "nixpkgs-review wip"`
- [ ] Tested execution of all binary files (usually in `./result/bin/`)
- [ ] Determined the impact on package closure size (by running `nix path-info -S` before and after)
- [ ] Ensured that relevant documentation is up to date
- [ ] Fits [CONTRIBUTING.md](https://github.com/NixOS/nixpkgs/blob/master/.github/CONTRIBUTING.md).

### CODE REVIEW

File: pkgs/tools/misc/hdl-dump/default.nix [hunk 1]
```diff
@@ -0,0 +1,36 @@
+{ stdenv
+, fetchFromGitHub
+, upx
+}:
+
+let
+  version = "20202807";
+  pname = "hdl-dump";
+in stdenv.mkDerivation {
+  inherit pname version;
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/hdl-dump/default.nix:10]:
```suggestion
stdenv.mkDerivation rec {
  pname = "hdl-dump";
  version = "20202807";
```

File: pkgs/tools/misc/hdl-dump/default.nix [hunk 2]
```diff
@@ -0,0 +1,36 @@
...
+
+  # playstation2 repo is outdated
```
Comment (SuperSandro2000) [inline pkgs/tools/misc/hdl-dump/default.nix:12]: Please add a link which one.

**Comment by SuperSandro2000** [comment]
Result of `nixpkgs-review pr 105514` run on x86_64-linux [1](https://github.com/Mic92/nixpkgs-review)
<details>
  <summary>1 package marked as broken and skipped:</summary>
  <ul>
    <li>hdl-dump</li>
  </ul>
</details>

File: pkgs/tools/misc/hdl-dump/default.nix (hunk 2 above)
Comment (AndersonTorres) [reply pkgs/tools/misc/hdl-dump/default.nix:12]: It is implict, but I will update it nonetheless.

**Comment by makefu** [comment]
@AndersonTorres  Thanks for picking up my contribution, i didn't have had much time to fix my PRs so i am glad you did :+1:

**Review by makefu — APPROVED** [review]

**Pull request merged**